.github/workflows/ci.yml

```diff
@@ -7,6 +7,7 @@ jobs:
   build-test:
     runs-on: windows-latest
     env:
+      api_keyman_com_host: http://127.0.0.1:8888
       api_keyman_com_mssql_pw: Password1!
```

Comment:
Ah yes, the ultimate in security for local unit testing.  /s

Not that it matters, given the setup.

Reply:
Yep. It's a local server instance which lives for a minute or two only, running isolated. It needs to be a "strong" password in order for the install to succeed, hence the `1!`  🤣